app/controllers/api/v2/config_templates_controller.rb

```diff
@@ -82,7 +82,7 @@ def destroy
       api :POST, "/config_templates/build_pxe_default", N_("Update the default PXE menu on all configured TFTP servers")
 
       def build_pxe_default
-        ::ActiveSupport::Deprecation.warn('GET method for build pxe default is deprecated. Please use POST instead.') if request.method == "GET"
+        Foreman::Deprecation.deprecation_warning("1.11", "GET method for build pxe default is deprecated. Please use POST instead") if request.method == "GET"
```

Comment:
Similar question to the v1, but slightly more acceptable as there's a replacement, but it seems to be that versioned APIs shouldn't be removed.